doc/configuration.markdown

```diff
@@ -124,4 +124,13 @@ which will act as the cache key generator:
 	end
 	
 For more options see the [Rack::Request documentation](http://rack.rubyforge.org/doc/classes/Rack/Request.html)
-	
+
+### `use_native_ttl`
+
+Passes on the expiration timestamp to the cache store. This may be necessary
+with some stores to keep them from filling up, e.g. if using a Redis backend and
```

Comment:
> you wrote your own redis backend
I didn't. This PR is mainly about making `Rack::Cache` work correctly with the [redis-store/redis-rack-cache](https://github.com/redis-store/redis-rack-cache) gem.

> want to link that in the readme
Sure thing.

> FYI I kind of want to get rid of all these silly backends and just support a common read/write interface and be one with it
Not sure what you mean? To my understanding `Rack::Cache` already has pluggable backends, as long as they're namespaced correctly, respond to a `.resolve(uri)` factory, and have instances that quack `#exists?`, `#read`, and `#write`?